<!DOCTYPE html>
<html lang="en">

<head>
<link href="https://fonts.googleapis.com/css?family=Didact+Gothic" rel="stylesheet">
<link rel="stylesheet" href="porecamp.css">
</head>

<body>

<!-- ---------------------------------------------------------------- -->
<h1>PoreCamp Australia 2017</h1>
<p id='blurb'>
A bootcamp to learn about operating the Oxford Nanopore MinION
</p>

<h2 id="Speakers">PoreCampAU Speakers</h2>

<!-- ---------------------------------------------------------------- -->

<!-- ---------------------------------------------------------------- -->

<h2 id="jq">Joshua Quick (UK)</h2>

<h3 id="jq_bio">Biography</h3>

After undergraduate study Joshua worked for Illumina Technologies as intergration scientist. Before </br>
commencing his his PhD at the University of Birmingham, UK. </br>
Joshua is a founder of PoreCamp along with his supervisor Nick Loman. </br>
Throughout his PhD, Joshua has been able to show many successful </br>
applications of real-time nanopore sequencing as demonstrated in the publications below. </br>


<a href="http://www.birmingham.ac.uk/schools/biosciences/staff/students/quick-joshua.aspx" > Biography source</a> </br>
<h3 id="jq_pub">Publications</h3>
<a href="http://gigadb.org/dataset/100102" > A reference bacterial genome dataset generated on the MinION &trade portable single-molecule nanopore sequencer. </a> </br>
<a href="http://www.nature.com/doifinder/10.1038/nature16996" > Real-time, portable genome sequencing for Ebola surveillance. </a> </br>
<a href="http://genomebiology.com/2015/16/1/114" > Rapid draft sequencing and real-time nanopore sequencing in a hospital outbreak of Salmonella. </a> </br>
<a href="http://www.nature.com/doifinder/10.1038/nmeth.3444" > A complete bacterial genome assembled de novo using only nanopore sequencing data. </a> </br>
<!-- ---------------------------------------------------------------- -->

<h2 id="de">David Eccles (NZ)</h2>

<h3 id="de_ab">Abstract</h3>

<i> After the stampede of sequenced reads, when the QC dust has settled and the mapping has been done, </br>
    it's time to explore the jungle of data provided by the MinION. You will be guided through the process of </br>
    pulling out signal from the noise, using MinION reads for variant detection and error correction, and shown </br>
    how these discoveries can be presented in a way that is easy to understand.</i> </br>

<h3 id="de_bio">Biography</h3>

David Eccles is a freelance bioinformatician who has worked (mostly remotely) for a number of different researchers  </br>
around the world. He is currently working out of an office at the <a href="http://www.malaghan.org.nz/news/miniature-dna-sequencer-on-trial/">Malaghan  </br>
Institute of Medical Research</a>, where he has been learning about immunology, flow cytometry,   </br>
and the all the things that can go wrong with the MinION sequencing device.  </br>  </br>

David's first placement as a conference speaker was talking about MinION sequencing at the 2015 NGS conference  </br>
in New Zealand, which led to a <a  href="http://www.youtube.com/watch?v=yTgZVVHJMlw">TEDxWellington </br>
talk</a> in March 2016 where he did sequencing on stage from a tomato source. </br>

<h3 id="de_pub">Publications</h3>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/26347715" > MinION nanopore sequencing of an influenza genome </a> </br>
<a href="http://dx.doi.org/10.12688/f1000research.7201.1">MinION Analysis and Reference Consortium: Phase 1 data release and analysis</a> </br>

<!-- ---------------------------------------------------------------- -->

</body>

</html>
